piff/basis_interp.py

```diff
@@ -449,7 +449,6 @@ def __init__(self, order, keys=('u','v'), max_order=None, use_qr=False, use_jax=
         self.use_jax = use_jax
 
         if not CAN_USE_JAX and self.use_jax:
-            logger.warning("JAX is not installed. Switching to numpy/scipy.")
```

Comment:
I think this line is worth keeping.  The better solution to the error you were getting is to add
```
logger = galsim.config.LoggerWrapper(logger)
```
before this.